stable/opsportal/values.yaml

```diff
@@ -49,3 +49,54 @@ opsportal:
           serviceName: traefik-kubeaddons-dashboard
           servicePort: 80
         path: /ops/portal/traefik
+
+
+###############################################################################
+#  Konvoy UI
+###############################################################################
+kommander:
+  image:
+    repository: mesosphere/kommander
+    tag: 1.60.0
+    pullPolicy: IfNotPresent
+  replicas: 1
+  logoutRedirectPath: /ops/landing
+  clusterPollingInterval: 3000
+  clientPollingInterval: 3000
+
+  # Mode must be either production|konvoy
+  mode: konvoy
+
+  displayName: Konvoy Cluster
+
+  createObservableClusterCRD: true
+
+  extraInitContainers:
+
+  resources:
+    requests:
+      memory: "128Mi"
+      cpu: "100m"
+    limits:
+      memory: "512Mi"
+      cpu: "2000m"
```

Comment:
nonblocking comment: do we have behavioral information that indicates it should be allowed to burst this high?